run.py

```diff
@@ -130,6 +130,9 @@ def create_accent_phrases(text: str, speaker_id: int) -> List[AccentPhrase]:
             return []
 
         utterance = extract_full_context_label(text)
+        if utterance.breath_groups == []:
```

Comment:
（ちょっと試してみたいので、suggestionして自分でOKしてみます）
```suggestion
        if len(utterance.breath_groups) == 0:
```